packages/apps/docs/.gitignore

```diff
@@ -17,3 +17,4 @@ src/pages/docs/chainweb/js-bindings
 
 next-env.d.ts
 
+broken-links.md
```

Comment:
not needed when we are not saving it in a MD file

Reply:
the file's only generated when the optional flag `--report-md` is present

Reply:
without a need for the md reports this should be gone